wiki/Client/Interface/Multiplayer/en.md

```diff
@@ -45,6 +45,8 @@ The listed match lobbies can be filtered using the options at the top left.
 
 Covering the middle of the screen is a list of available matches.
 
+Most matches have light white backgrounds, indicating that they were created normally in [this way](#creating-a-new-game). While there are also ones with purple backgrounds, which were **tournament matches** created by using `!mp make` or `!mp makeprivate`.
```

Comment:
avoid links that say "this link" or "here", because at a glance it's less obvious what it's linking to. the writing should be the same whether there is a link or not.

```suggestion
Most matches have light white backgrounds, indicating that they were [created normally through the game UI](#creating-a-new-game). There are also ones with purple backgrounds, which are *tournament matches* created and managed using [lobby management chat commands](/wiki/osu!_tournament_client/osu!tourney/Tournament_management_commands) such as `!mp make` or `!mp makeprivate`.
```